### PR TITLE
DropDown-based editors: make dropDownOptions public

### PR DESCRIPTION
**Pull request by ksercs**

### CODE REVIEW

File: js/ui/drop_down_editor/ui.drop_down_editor.d.ts [hunk 1]
```diff
@@ -39,6 +43,14 @@ export interface dxDropDownEditorOptions<T = dxDropDownEditor> extends dxTextBox
      * @public
      */
     applyValueMode?: 'instantly' | 'useButtons';
+    /**
+     * @docid dxDropDownEditorOptions.dropDownOptions
+     * @type dxPopupOptions
+     * @default {}
+     * @prevFileNamespace DevExpress.ui
+     * @hidden
```
Comment (DokaRus) [inline js/ui/drop_down_editor/ui.drop_down_editor.d.ts]: Could we change it to
```suggestion
     * @public
```
and remove redefinition of the `dropDownOptions`?

Comment (ksercs) [reply js/ui/drop_down_editor/ui.drop_down_editor.d.ts]: fixed

**Review by DokaRus — DISMISSED** [review]

**Review by DokaRus — DISMISSED** [review]

File: js/ui/lookup.d.ts [hunk 2]
```diff
@@ -378,6 +386,11 @@ export interface dxLookupOptions extends dxDropDownListOptions<dxLookup> {
      * @public
      */
     usePopover?: boolean;
+    /**
+     * @docid dxLookupOptions.dropDownOptions
+     * @type dxPopupOptions | dxPopoverOptions
+     */
+    dropDownOptions?: {};
```
Comment (San4es) [inline js/ui/lookup.d.ts]: Why not `dropDownOptions?: dxPopupOptions | dxPopoverOptions`? @IlyaKhD our tooling accepts such cases?

Comment (ksercs) [reply js/ui/lookup.d.ts]: You're right. Fixed

**Review by San4es — APPROVED** [review]

**Review by DokaRus — APPROVED** [review]

**Pull request merged**